docs/tutorial/tutorial-0.rst

```diff
@@ -92,6 +92,10 @@ Next, install Toga into your virtual environment:
 
       (venv) C:\...>pip install --pre toga
 
+.. note:: Python 3.8 Limitations on Windows
+
+    You can use any stable version of Python 3.5, 3.6, 3.7. We would advice against using Python 3.8 for Windows as some dependecies aren't avaliable for Python 3.8 yet.
```

Comment:
3 typos:
advice -> advise
dependecies -> dependencies
avaliable -> available

It's also generally considered good grammar to avoid ending a sentence in a proposition; I'd suggest rephrasing the end of the sentence as "are not currently available for Python 3.8".

Reply:
Thank you @freakboy3742 for your feedback.
I have made the required changes.